### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by felixodette**
- Understand the concept of state management.
 - Use store, actions and reducers in React.
- Connect React and Redux.
- Connect an existing API via Redux and use the data to display something in a UI.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/2kLjGe8d2F6O626umTAc/books';
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/kHtQHkSgi4TSFEWzhSpl/books';
 
 // Actions
 const ADD_BOOK = 'bookstore/booksReducer/ADD_BOOK';
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [ ] When I load your app, after a while, it starts logging errors related to fetching the books. The errors stop for a while, then continue again. Kindly fix this.

  felix-fetch

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @felixodette,

Good job so far!
✔️ Good documentation
✔️ Linters are passing
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] I was unable to add a category. When I type in the Category input, it reflects on the author input. This is because of how you set up this function. Kindly fix this. I believe you meant: `setCategory`, not `setAuthor`

  felix-1

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @felixodette,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by felixodette** [comment]
Thanks for the quick review!

**Pull request merged**